eyeshade/workers/reports.js

```diff
@@ -815,6 +816,153 @@ const date2objectId = (iso8601, ceilP) => {
                        (ceilP ? 'ffffffffffffffff' : '0000000000000000'))
 }
 
+/**
+ * A referral statement consists of entries describing accrued earnings from referrals,
```

Comment:
technically, the accounting term "accrued earnings" is probably correct here (i'm not an accountant); however, i'm a little concerned about using accounting terms-of-art in code with an actual accountant reviewing it. i prefer to use generic terms to avoid any confusion by domain experts...